### PR TITLE
Docker Updates

### PR DESCRIPTION
**Pull request by Fhrozen**
Updates on docker to fit the installation of ESpnet.
This will also fix PR #2508.

### CODE REVIEW

**Comment by Fhrozen** [comment]
Finished, containers are being pushed.

File: docker/prebuilt/devel/Dockerfile [hunk 1]
```diff
@@ -18,21 +18,25 @@ WORKDIR /espnet/tools
...
 # Replace nvidia-smi for nvcc because docker does not load nvidia-smi
-RUN if [ -z "$( nvcc -V )" ]; then \
+RUN if [ -z "$( which nvcc )" ]; then \
         echo "Build without CUDA" && \
-        MY_OPTS="CUPY_VERSION="; \
+        MY_OPTS='CUPY_VERSION=""  TH_VERSION=1.6.0'; \
     else \
         echo "Build with CUDA" && \
+        # Docker containers cannot load cuda libs during build.
+        # So, they checks on cuda package are disable.
```
Comment (sw005320) [inline docker/prebuilt/devel/Dockerfile:27]:
```suggestion
        # So, their checks on cuda packages are disabled.
```

File: docker/prebuilt/local/Dockerfile [hunk 2]
```diff
@@ -22,24 +22,27 @@ RUN cd espnet && \
...
-RUN if [ -z "$( nvcc -V )" ]; then \
+RUN if [ -z "$( which nvcc )" ]; then \
         echo "Build without CUDA" && \
-        MY_OPTS="CUPY_VERSION="; \
+        MY_OPTS='CUPY_VERSION=""  TH_VERSION=1.6.0'; \
     else \
         echo "Build with CUDA" && \
+        # Disable cupy test
+        # Docker build does not load libcuda.so.1
+        # So, they checks on cuda package are disable.
```
Comment (sw005320) [inline docker/prebuilt/local/Dockerfile:33]:
```suggestion
        # So, their checks on cuda packages are disabled.
```

**Comment by sw005320** [comment]
Many thanks!

**Pull request merged**